WalletWasabi/Tor/Http/ClearnetHttpClient.cs

```diff
@@ -9,12 +10,10 @@ namespace WalletWasabi.Tor.Http
 	/// HTTP client implementation based on .NET's <see cref="HttpClient"/> which provides least privacy for Wasabi users,
 	/// as HTTP requests are being sent over clearnet.
 	/// </summary>
-	public class ClearnetHttpClient : IHttpClient
+	/// <remarks>Inner <see cref="HttpClient"/> instance is thread-safe.</remarks>
+	public class ClearnetHttpClient : IRelativeHttpClient
```

Comment:
What is the difference between this and BackendHttpClient ?

Reply:
There is no real difference. I would love to have only `BackendHttpClient` but I need to refactor quite a lot of code to get there and this `IRelativeHttpClient` helps me move it forward. So it is rather intermediate step towards that goal.

Reply:
ACK